apps/minifront/src/components/swap/lp-positions.tsx

```diff
@@ -0,0 +1,27 @@
+import { Card } from '@penumbra-zone/ui/components/ui/card';
+import { GradientHeader } from '@penumbra-zone/ui/components/ui/gradient-header';
+import { useOwnedPositions } from '../../state/swap/lp-positions.ts';
+import { bech32mPositionId } from '@penumbra-zone/bech32m/plpid';
+
+// TODO: Ids are not sufficient in taking action on these
+//       Required to move forward with this: https://github.com/penumbra-zone/penumbra/pull/4837
+export const LpPositions = () => {
+  const { data, error } = useOwnedPositions();
+
+  return !data?.length ? (
+    <div className='hidden xl:block'></div>
+  ) : (
+    <Card layout>
+      <GradientHeader layout>Limit orders</GradientHeader>
+      {error ? <div>❌ There was an error loading your limit orders</div> : undefined}
```

Comment:
nitpick: this can be simplified to 
```suggestion
      {error && <div>❌ There was an error loading your limit orders</div>}
```

Reply:
Have to coerce to a boolean or I get `TS2322: Type unknown is not assignable to type ReactNode`. Also will include the error in the ui:

```ts
{!!error && <div>❌ There was an error loading your limit orders: ${String(error)}</div>}
```